### DISCUSSION TITLE
JavaScript or Python backend?

### DISCUSSION THREAD

**Discussion by InvalidLenni** [AntiScamBot/Anti-Scam-Link-List#17] [Ideas] [answered]
I start to write the backend for the antiscam bot website tomorrow, so should I use python or JavaScript for the backend?

**Comment by 2Lost4This** [comment, marked as answer]
> I start to write the backend for the antiscam bot website tomorrow, so should I use python or JavaScript for the backend?

whatever you want, would like Js more because i can understand it but whats easier for you

**Reply by InvalidLenni** [reply]
Then in JavaScript because you can understanf js more as py

**Reply by InvalidLenni** [reply]
Oops, i mean understand

**Reply by 2Lost4This** [reply]
awsome

**Reply by InvalidLenni** [reply]
hm, I think about to rewrite the current backend to python because understand more python

**Reply by 2Lost4This** [reply]
> hm, I think about to rewrite the current backend to python because understand more python

As you want

**Reply by InvalidLenni** [reply]
Yeah idk

**Comment by InvalidLenni** [comment]
I unmark the answer rn for users opinion.
